### PR TITLE
#4821 [Manager] After created workspace, the settings of the each services should be backed up to the workspace

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4821)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by saivirtue** [comment]
retry manager-e2e

File: ohara-manager/client/src/store/epics/broker/createBrokerEpic.js [hunk 1]
```diff
@@ -24,21 +24,36 @@ import {
...
+            schema.broker,
+            normalizedData.entities,
+          ),
+          // the workspace name of current object should be as same as the broker name
+          name: values.name,
+          group: GROUP.WORKSPACE,
+        }),
+        actions.createBroker.success(normalizedData),
+      ]),
+    ),
```
Comment (saivirtue) [inline ohara-manager/client/src/store/epics/broker/createBrokerEpic.js:56]: /cc @eechih 
Please help me review this part :)
I use the `denormalize` API to de-structure the normalize data and trigger the update action

File: ohara-manager/client/src/store/epics/broker/createBrokerEpic.js [hunk 2]
```diff
@@ -24,21 +24,36 @@ import {
...
 import { getId } from 'utils/object';
-import { LOG_LEVEL } from 'const';
+import { LOG_LEVEL, GROUP } from 'const';
 
 export const createBroker$ = values => {
   const brokerId = getId(values);
   return defer(() => brokerApi.create(values)).pipe(
     map(res => res.data),
     map(data => normalize(data, schema.broker)),
     map(normalizedData => merge(normalizedData, { brokerId })),
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/broker/createBrokerEpic.js:41]:
```suggestion
    map(res => res.data),
    switchMap(data => 
        const normalizedData = merge(normalize(data, schema.broker), { brokerId });
        return from([
            actions.updateWorkspace.trigger({
                broker: data,
                // the workspace name of current object should be as same as the broker name
                name: values.name,
                group: GROUP.WORKSPACE,
            }, 
            actions.createBroker.success(normalizedData),
        ])
     ),
```

File: ohara-manager/client/src/store/epics/broker/createBrokerEpic.js (hunk 1 above)
Comment (eechih) [reply ohara-manager/client/src/store/epics/broker/createBrokerEpic.js:56]: `denormalize` 感覺很冗餘 🤔

**Pull request merged**